vhdbuilder/packer/install-dependencies.sh

```diff
@@ -46,9 +46,39 @@ systemctlEnableAndStart update_certs.timer || exit 1
 systemctlEnableAndStart ci-syslog-watcher.path || exit 1
 systemctlEnableAndStart ci-syslog-watcher.service || exit 1
 
+if [[ ${UBUNTU_RELEASE} == "18.04" && ${ENABLE_FIPS,,} == "true" ]]; then
```

Comment:
debugged FIPS pulling in non-FIPS kernel only to realize the base _is_ non-FIPS (correctly)

moving this to the beginning, once we have 1es/network lockdown, we should installFips -> reboot -> rest of vhd build runs against FIPS image

will also allow us to prune unused kernels later.